bin/image-create-gce.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+GCE_ACCOUNT_FILE_PATH=${GCE_ACCOUNT_FILE_PATH:-1}
+if [ -z "$GCE_ACCOUNT_FILE_PATH" ]; then
+  echo "env GCE_ACCOUNT_FILE_PATH must set or be the first argument for this script"
+  exit 1
+fi
+
+IMG_VERSION=${IMG_VERSION:-"v040b1"}
+IMAGE_NAME="kubenow-$IMG_VERSION"
+
+echo "Login"
+gcloud auth activate-service-account --key-file="$GCE_ACCOUNT_FILE_PATH"
+
+project_id=$(jq -r .project_id < "$GCE_ACCOUNT_FILE_PATH")
+gcloud config set project "$project_id"
+
+echo "Check if image exists already"
+image_status="$(gcloud compute images list)"
+image_name=$(echo "$image_status" | grep  "\b$IMAGE_NAME\s" || true)
+if [ -z "$image_name" ]; then
+
+  SECONDS=0
```

Comment:
There is so much code for the spinner please use something already available on the internet https://github.com/tlatsas/bash-spinner, or please define your spinner in a separated file.

Reply:
I did look into already existing spinner functions - also that very one from github, but I think code got much more complicated calling external function.
Currently the spinner also includes an elapsed time message to user, and the spinner is only about 2 lines of code. Formating time is 3 lines:) 
I did elaborate the code comment of the loop to make it much clearer

Reply:
Looks good!